students/skowronski_rafal/lesson_10_organizing_files/deleting_unneeded_files.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+
+def get_search_directory():
+    return os.path.abspath(os.sep)
+
+
+def find_files(search_directory, predicate):
+    matched_files = []
+    for current_dir, sub_dirs, file_names in os.walk(search_directory):
+        files_portion = [
```

Comment:
It's ok, but this list comprehension is not clear. Let's think how to organize code in separate functions